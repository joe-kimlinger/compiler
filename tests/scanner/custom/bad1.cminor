x: string "hello this is" not a string"
