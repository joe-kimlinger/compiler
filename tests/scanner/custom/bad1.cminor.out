IDENTIFIER x
COLON
STRING
STRING_LITERAL hello this is
IDENTIFIER not
IDENTIFIER a
STRING
